### PR TITLE
Applied feedback from minimalism workshop sessions

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://trello.com/c/mwDac5Q2/443-apply-minimalism-feedback-to-the-managing-environment-variables-topic

- Pointed users to CLI reference instead of including CLI content in the topic
- Removed the Overview section that had redundant information and a poor heading
- Rearranged the sections based on user process

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @bfallonf PTAL

File: dev_guide/environment_variables.adoc [hunk 1]
```diff
@@ -11,75 +11,30 @@
...
-
-== CLI Interface
-
-{product-title} provides the `oc set env` command to set or unset environment
-variables for objects that have a pod template, such as replication controllers
-or deployment configurations. It can also list environment variables in pods or
-any object that has a pod template. This command can also be used on
-`*BuildConfig*` objects.
+[[using-the-oc-set-env-command]]
+== Using the oc set env Command
```
Comment (bfallonf) [inline dev_guide/environment_variables.adoc]: @ahardin-rh Can I suggest a title change? "Setting and Unsetting Environment Variables"

**Comment by bfallonf** [comment]
One comment. Looks good otherwise :star2:

**Comment by adellape** [comment]
Noice. :headphones: :

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../dev_guide/environment_variables.adoc#dev-guide-environment-variables[Managing Environment Variables]
|Removed redundant information and CLI reference material; rearranged sections  to match user process.
%

**Pull request merged**